docs/source/release.rst

```diff
@@ -13,6 +13,7 @@ Release Notes
 * :feature:`2126` Add translation rules for isnull() and notnull() for pyspark backend
 * :feature:`2062` Implement read_csv for omniscidb backend
 * :feature:`2097` Date, DateDiff and TimestampDiff implementations for OmniSciDB
+* :bug:`2170` Fix millisecond issue for OmniSciDB, MySQL, PostgreSQL and Pandas backends
```

Comment:
can you add all of the issues numbers here (same issue if fine)

Reply:
I added all the issues number near each backend name.